api/coreservice.go

```diff
@@ -496,7 +496,7 @@ func (core *coreService) PendingNonce(addr address.Address) (uint64, error) {
 		return nonce, err
 	}
 	g := core.Genesis()
-	if nonce > 1 || !g.IsSumatra(core.TipHeight()) {
+	if nonce > 1 || !g.IsToBeEnabled(core.TipHeight()) {
```

Comment:
Should use a feature flag here?

Reply:
coreservice has no access to feature flag, instead use `g.Isxxx()`. Other checks like enforce chain ID also the same